django_cloud_tasks/connection.py

```diff
@@ -48,9 +51,12 @@ def __get__(self, obj, cls):
 class GoogleCloudClient(object):
     @cached_property
     def client(self):
-        client = googleapiclient.discovery.build('cloudtasks', 'v2beta3',
-                credentials=DCTConfig.google_cloud_credentials(), 
-                cache=DiscoveryCache())
+        client = googleapiclient.discovery.build(
```

Comment:
you want to replace this client with the new version.

Reply:
fixed